internal/cmd/controller/grutil/resourcekey.go

```diff
@@ -40,24 +40,29 @@ func fromResourceKey(ctx context.Context, c client.Client, namespace, name strin
 		errors    []string
 	)
```

Comment:
Function-level documentation needs to be updated here, since this function no longer lists bundles.
And the note on resources listed in bundles should be edited or removed.

Reply:
True. It's updated